src/mainviews/create-job.tsx

```diff
@@ -295,6 +298,7 @@ export function CreateJob(props: ICreateJobProps): JSX.Element {
             name="computeType"
             id={`${formPrefix}computeType`}
             onChange={handleSelectChange}
+            environmentList={environmentList}
             environment={props.model.environment}
             initialValue={props.model.computeType || ''}
```

Comment:
Remove the default value of `''` here since we will be setting the default as soon as environment gets selected.